pkgs/development/python-modules/shapely/default.nix

```diff
@@ -36,7 +36,7 @@ buildPythonPackage rec {
   # Disable the tests that improperly try to use the built extensions
   checkPhase = ''
     rm -r shapely # prevent import of local shapely
-    py.test tests
+    py.test -k 'not test_svg' tests
```

Comment:
Can you instead of disabling the test import the following patch to the patches list?

```
    # included in next release.
    (fetchpatch {
      url = "https://github.com/Toblerity/Shapely/commit/ea5b05a0c87235d3d8f09930ad47c396a76c8b0c.patch";
      sha256 = "sha256-egdydlV+tpXosSQwQFHaXaeBhXEHAs+mn7vLUDpvybA=";
    })
```

Reply:
Done.